x-pack/plugins/apm/public/components/shared/truncate_with_tooltip/index.tsx

```diff
@@ -45,3 +45,9 @@ export function TruncateWithTooltip(props: Props) {
     </TooltipWrapper>
   );
 }
+
+export function TruncateWithoutTooltip(props: Props) {
```

Comment:
can you please help me understanding why we need another component without a tooltip?

Reply:
@kpatticha We need another component here because earlier truncation was happening at group level. Now, I have modified it in a way that truncation would be applied on item level.